src/modules/users/components/settings-card/settings-card.tsx

```diff
@@ -0,0 +1,39 @@
+import { useNavigate } from 'react-router-dom';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { ROUTES } from '../../constants';
+import { settingsCardStyles as styles } from './settings-card.styles';
+
+type Props = {
+  icon: string;
+  title: string;
```

Comment:
Give a type to the title. It should be a key of the maps object below.

Reply:
fixed